src/azure-cli/azure/cli/command_modules/profile/tests/latest/test_profile.py

```diff
@@ -28,10 +28,12 @@ def test_account_show(self):
             assert result['user']['name'] == MOCKED_USER_NAME
 
     def test_list_locations(self):
-        result = self.cmd('account list-locations').get_output_in_json()
+        result = self.cmd('account list-locations --include-extended-locations').get_output_in_json()
         assert isinstance(result, list)
         # Verify there is an item with displayName 'East US'.
         assert any('East US' == loc['displayName'] for loc in result)
+        assert any('metadata' in loc for loc in result)
+        assert any('availabilityZoneMappings' in loc for loc in result)
```

Comment:
`metadata` and `availabilityZoneMappings` are the extended locations you want to check, right?

Reply:
`metadata` and `availabilityZoneMappings` are parameters checked in response.
![image](https://github.com/Azure/azure-cli/assets/37168047/faabc0ec-034c-40ab-ab5d-f2f17a4067e1)